appveyor.yml

```diff
@@ -89,17 +108,17 @@ install:
     - cmd: conda update --yes --quiet conda
 
     - cmd: conda install --yes --quiet obvious-ci conda-build-all=1.0.0
-    - cmd: conda install --yes --quiet conda-forge-build-setup
+    - cmd: conda install --yes --quiet conda-forge-build-setup=3.*
```

Comment:
We need to investigate why it is failing. I am not sure you need https://github.com/conda-forge/staged-recipes/commit/f65966a172dad4c96e18335cfacea911981db6af